index.html

```diff
@@ -167,78 +165,88 @@
         <dd>
           <p>This method operates identically to <code>getUserMedia</code>,
-          expect that it acquires media from output devices.</p>
+          except that it acquires media from output devices.</p>
         </dd>
       </dl>
     </section>
 
     <section>
-      <h2>Constraining Display Surface Selection</h2>
+      <h2 id="constraints">Constraining Display Surface Selection</h2>
 
-      <p>Two constraints are defined that allow an application to limit the
-      display surfaces that are selected.  This only allows an application to
-      control the type of surface that is selected, not to identify a specific
-      surface.</p>
+      <p>The <code><a>getDisplayMedia</a></code> function permits the use of
+      constraints in the same way that constraints are defined
+      for <code><a>getUserMedia</a></code>.  However, these constraints MUST NOT
+      constrain the selection of choices that are presented to users in the same
+      way.</p>
 
-      <p>The <code><a
-      href="#widl-MediaTrackConstraintSet-displaySurface">displaySurface</a></code>
-      constraint allows an application to express a preference or requirement
-      for the type of <a>display surface</a> that is acquired.</p>
-
-      <p>The <code><a
-      href="#widl-MediaTrackConstraintSet-logicalSurface">logicalSurface</a></code>
-      constraint allows an application to express a preference or requirement to
-      capture the <a>logical display surface</a>, rather than the <a>visible
-      display surface</a>.</p>
-
-      <dl class="idl" title="partial dictionary MediaTrackConstraintSet">
-        <dt>ConstrainDOMString displaySurface</dt>
-
-        <dd>
-          <p>What type of <a>display surface</a> to capture.  This assumes
-          values from the <code><a>OutputCaptureSurfaceType</a></code>
-          enumeration.</p>
-        </dd>
-
-        <dt>ConstrainBoolean logicalSurface</dt>
-
-        <dd>
-          <p>A value of <code>true</code> requests or requires capture of a
-          <a>logical display surface</a>; a value of <code>false</code> requests
-          or requires capture of a <a>visible display surface</a>.</p>
-        </dd>
-      </dl>
+      <p>Once a <a>display surface</a> has been selected, constraints apply to
+      the <a>MediaStreamTrack</a> instances that are returned.  This allows for
+      changes such as adjustments to frame rate or resolution, or other
+      constraints.</p>
 
       <section>
-        <h2><code>OutputCaptureSurfaceType</code></h2>
+        <h2>New Constraints for Captured Display Surfaces</h2>
+        <p>Two constraints are defined that allow an application to observe
+        properties of the selected <a>display surface</a>.  Since the source of
+        media cannot be changed after a <code>MediaStreamTrack</code> has been
+        returned, these constraints cannot be changed by an application.</p>
+
```

Comment:
This intro paragraph is not consistent with the description of the "logicalSurface" constraint below, since that is a preference, not just an observable property.